.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: node_js
+node_js:
+  - '4'
+  - '6'
+script:
+  - npm run coverage
+  - npm run codecov
+notifications:
+  email:
+    on_success: change
+    on_failure: change
```

Comment:
I'm not sure this is really useful at the moment. As far as I know you need to provide the recipients. We usually use HipChat for build notifications. I would use that instead.

https://docs.travis-ci.com/user/notifications/#HipChat-notification